investor_wallet_platform_base/views/invoice_view.xml

```diff
@@ -27,8 +27,7 @@
         <field name="inherit_id" ref="account.view_account_payment_invoice_form"/>
         <field name="arch" type="xml">
             <field name="journal_id" position="attributes">
-                <attribute name="invisible">True</attribute>
-            </field>
+@            </field>
```

Comment:
Clone on same line?
I understand this one non-action line is kept because this view has been created and can't be left empty?

Reply:
No, it's just a mistake :p I'll put it back in :p thx!